geocity/apps/submissions/views.py

```diff
@@ -137,6 +137,27 @@ def get_submission_for_prolongation(user, submission_id):
     return submission
 
 
+def get_submission_service_fee(user, submission_id):
+    allowed_statuses = models.Submission.SERVICE_FEES_STATUSES
+    submission = get_submission_for_user_or_404(
+        user,
+        submission_id,
+        statuses=allowed_statuses,
+    )
+
+    return submission
```

Comment:
Move to model instead ?

Reply:
Not crictical for the moment. The feature is subject to near future changes.